package.json

```diff
@@ -98,5 +98,6 @@
     "vite": "^5.2.10",
     "vitest": "1.6.0",
     "vue": "^3.4.27"
-  }
+  },
+  "stableVersion": "0.3.1"
```

Comment:
Yarn seems to do this automatically when we publish an alpha, so that it knows what the most recent stable version was if you decide to publish another? Seems harmless, and will likely disappear on the next stable version.